README.md

```diff
@@ -48,6 +49,28 @@ and run `esy` in the project root. To install [Esy](https://esy.sh), `npm
 install -g esy` should cover most workflows. If you have NodeJS / `npm`
 available, `npx esy` is even shorter.
 
+### [Opam](https://opam.ocaml.org/)
+
+You can also install melange with `opam`. First, have a look at the `esy.json` file to see if any dependencies are expected to be at particular commits. For example, you may need to pin `reason` to a particular version:
+
+    opam pin reason https://github.com/reasonml/reason.git#SOME_GIT_SHA_FROM_THE_ESY_JSON -y -n
+
+Then pin melange to the version on github (there is currently no officially released verison of melange in the opam repositories themselves):
+
+    opam pin melange https://github.com/melange-re/melange.git
+
+You should now be able to run melange from your new switch:
+
+    opam exec -- mel --help
+
+Melange also has some runtime dependencies that need to be symlinked into `node_modules` to be discoverable by bundlers like webpack:
+
+    rm -rf node_modules/melange
+    mkdir -p node_modules/melange
+    ln -sfn $(opam var prefix)/lib/melange node_modules/melange/lib
```

Comment:
This is a little unfortunate. I'd love to find a good solution for this. it might be enough to set `NODE_PATH=$(opam var prefix)/lib/melange` or something. @actionshrimp this is not actionable for you, just noting it down.

Reply:
One other thing I tried to do originally was to install melange with npm itself (as it has a package.json) - I guess this could be a way to get the runtime dependencies down (although the files didn't end up in the correct place this way either, the package.json probably needs some changes to support this).

This would however then introduce a new problem of keeping the ocaml side in sync with the node side which isn't ideal either, but thought I'd mention it in case it conjoured up any other ideas.